_posts/news/2022-04-04-announcing-scalajs-1.10.0.md

```diff
@@ -0,0 +1,95 @@
+
+Read on for more details.
+
+<!--more-->
+
+## Getting started
+
+If you are new to Scala.js, head over to [the tutorial]({{ BASE_PATH }}/tutorial/).
+
+If you need help with anything related to Scala.js, you may find our community [in `#scala-js` on Discord](https://discord.com/invite/scala) and [on Stack Overflow](https://stackoverflow.com/questions/tagged/scala.js).
+
+Bug reports can be filed [on GitHub](https://github.com/scala-js/scala-js/issues).
+
+## Release notes
+
+If upgrading from Scala.js 0.6.x, make sure to read [the release notes of Scala.js 1.0.0]({{ BASE_PATH }}/news/2020/02/25/announcing-scalajs-1.0.0/) first, as they contain a host of important information, including breaking changes.
+
+This is a **minor** release:
+
+* It is backward binary compatible with all earlier versions in the 1.x series: libraries compiled with 1.0.x through 1.9.x can be used with 1.10.0 without change.
+* Despite being a minor release, 1.10.0 is forward binary compatible with 1.8.x and 1.9.x. It is *not* forward binary compatible with 1.7.x. Libraries compiled with 1.10.0 can be used with 1.8.x and 1.9.x but not with 1.7.x or earlier.
+* It is *not* entirely backward source compatible: it is not guaranteed that a codebase will compile *as is* when upgrading from 1.9.x (in particular in the presence of `-Xfatal-warnings`).
+
+As a reminder, libraries compiled with 0.6.x cannot be used with Scala.js 1.x; they must be republished with 1.x first.
+
+## Fixes with compatibility concerns
+
+### `java.util.UUID.randomUUID()` now depends on `java.security.SecureRandom`, which is provided by external libraries
+
+Until Scala.js 1.9.x, `java.util.UUID.randomUUID()` was implemented using `java.util.Random`, which was not cryptographically secure, and was therefore a security issue.
+To address that, Scala.js 1.10.x uses `java.security.SecureRandom` in its implementation instead.
+
+However, `SecureRandom` cannot be implemented on all JavaScript environments, as ECMAScript, per se, lacks any primitive to access a good source of entropy.
+While it can be implemented on top of browser APIs or Node.js APIs, we have no decent fallback for other cases.
+Not fully implementing it correctly for all platforms would go against our policy for the standard library, i.e., that linking code must be correct.
+Therefore, Scala.js core does not provide any implementation of `java.security.SecureRandom`.
+
+This means that code that was previously calling `java.util.UUID.randomUUID()` will now fail to link.
+To preserve binary compatibility, we introduce two variants of a library that provides `java.security.SecureRandom`:
+
+* [`scalajs-java-securerandom`](https://github.com/scala-js/scala-js-java-securerandom) provides a *correct*, cryptographically secure implementation of `java.security.SecureRandom`, but relies on the Node.js `crypto` package or the Web Crypto API `crypto.getRandomValues` to be available (e.g., in browsers).
```

Comment:
s/package/module/ ? IIUC a package in this context would mean something you need to install via npm. Which is not necessary.